<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindSettings</class>
 <widget class="QDialog" name="FindSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="g_searchType">
       <property name="title">
        <string>Find</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="r_ascii">
          <property name="text">
           <string>&amp;ASCII String</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="r_hex">
          <property name="text">
           <string>&amp;HEX Bytes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="r_regex">
          <property name="text">
           <string>&amp;Regular Expression</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_caseSensitive">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="l_searchFor">
         <property name="text">
          <string>Search for:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="e_searchFor"/>
       </item>
       <item>
        <widget class="QPushButton" name="b_validationCheck">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="c_buttonBox">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
